sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2021-02-08)
+### Breaking Changes
+- Deleted CallingApplicationIdentifier
+- Deleted getId function from CommunicationIdentifier class
+
+### Added
+- New MicrosoftTeamsUserIdentifier constructor that takes a non-null CommunicationCloudEnvironment parameter
+- Added class CommunicationCloudEnvironment
+- Added class CommunicationCloudEnvironmentModel
+- Added class CommunicationIdentifierSerializer
+- Added class CommunicationIdentifierModel
+- Added class MicrosoftTeamsUserIdentifierModel
+- Added class PhoneNumberIdentifierModel
+- Added class CommunicationUserIdentifierModel 
```

Comment:
```suggestion
### New Features
- Added a new constructor to `MicrosoftTeamsUserIdentifier` that takes a non-null CommunicationCloudEnvironment parameter.
- Added class `CommunicationCloudEnvironment`.
- Added class `CommunicationCloudEnvironmentModel`.
- Added class `CommunicationIdentifierSerializer`.
- Added class `CommunicationIdentifierModel`.
- Added class `MicrosoftTeamsUserIdentifierModel`.
- Added class `PhoneNumberIdentifierModel`.
- Added class `CommunicationUserIdentifierModel`.
```

Reply:
But another guideline says: "Added".

https://azure.github.io/azure-sdk/policies_releases.html#changelog-guidance

Reply:
I'll ignore the changelog-guidance and follow the release notes guideline. :-)

Reply:
That's interesting, I need to talk to @czubair about this, I've seen a few libraries use the release notes guidelines instead, mostly due to automated scripts that picks up the changelog to populate each month's release notes.